### PR TITLE
you wouldnt believe it but i am a native english speaker

### PR DESCRIPTION
**Pull request by sourTaste000**
Changes made on Bedrock Linux running on Manjaro Kernel

### CODE REVIEW

File: src/routes/index.svelte.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Manjaro has a track record of pretty poor security.
 
 ### SSL Certificates
 
-[Manjaro have let their certificates expire twice!](https://redd.it/4inrut/)
+[Manjaro has let their certificates expire twice!](https://redd.it/4inrut/)
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:24]: have is correct, this is plural...?

File: src/routes/index.svelte.md [hunk 2]
```diff
@@ -48,8 +48,8 @@ I have used both Manjaro and Arch for a while, and have ironically had fewer pro
 You've probably seen across their resources that Manjaro holds Arch packages behind for two
 weeks. Stability is stated as the reason for this, but that doesn't make much sense.
 
-In practice, what this actually means is that software upgrades reach you later for seemingly
-no reason. That means getting updates -- new features, security patches, bug fixes -- two whole weeks late. Aside from the obvious downside this brings, it also causes a more insidious issue.
+In practice, what this means is that software upgrades reach you later for seemingly
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:51]: what does removing actually do lol

File: src/routes/index.svelte.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ blocked again. This may have been the cause for the day's earlier outage.
 
 * https://gitlab.manjaro.org/applications/pamac/-/issues/1135
 
-While these incidents were in no way intentional, it highlights the poor QA testing that Manjaro performs. This has happened on two seperate occasions in less than two years.
+While these incidents were in no way intentional, it highlights the poor QA testing that Manjaro performs. This has happened on two separate occasions in less than two years.
```
Comment (EmeraldSnorlax) [inline src/routes/index.svelte.md:88]: this is actually a typo

**Review by EmeraldSnorlax — CHANGES_REQUESTED** [review]
`seperate` is the only typo here?

File: src/routes/index.svelte.md (hunk 1 above)
Comment (d1snin) [reply src/routes/index.svelte.md:24]: This is a typo:
- [x] Manjaro is singular
- [x] Third-person narrative is used

Comment (EmeraldSnorlax) [reply src/routes/index.svelte.md:24]: manjaro is plural; im referring to the organisation of people, not the distro

Comment (EmeraldSnorlax) [reply src/routes/index.svelte.md:24]: actually im dumb as fuck

**Comment by EmeraldSnorlax** [comment]
im dumb as fuck

File: src/routes/index.svelte.md (hunk 1 above)
Comment (d1snin) [reply src/routes/index.svelte.md:24]: > manjaro is plural in this context; im referring to the organisation of people, not the distro

Kinda makes sense :trollface:

**Pull request merged**